documentation/specs/Banner.md

```diff
@@ -0,0 +1,46 @@
+# Banner Component Specification
+
+## Overview
+
+The `Banner` component displays a prominent message above the navigation bar at the top of the page that cannot be dismissed.
+
+### Use Cases
+
+The primary use case of the `Banner` component is to inform the user of promotions and other offerings on the marketing (logged out) site. **Not** to be used to call attention to the user as a result of interacting with the UI.
```

Comment:
**question:** is a banner intended to *only* be used on the marketing site? like exclusively? or does that just happen to be the more common pattern?

i'm wondering how strong of a scope stance we need to take here in the use cases for it

Reply:
to my understanding, its scope is limited to the marketing site, but on second thought, i do think perhaps its not really necessary for us to take that sort of stance for the docs